src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/KinesisClientLibConfiguration.java

```diff
@@ -466,6 +486,21 @@ public KinesisClientLibConfiguration withKinesisEndpoint(String kinesisEndpoint)
     }
 
     /**
+     * @param dynamoDBEndpoint DynamoDB endpoint
+     * @return KinesisClientLibConfiguration
+     */
+    public KinesisClientLibConfiguration withDynamoDBEndpoint(String dynamoDBEndpoint) {
+        this.dynamoDBEndpoint = dynamoDBEndpoint;
+
+        // call .setEndpoint() on the current dynamoDBClientConfig
+        ClientConfiguration config = this.dynamoDBClientConfig;
+        config.setEndpoint(dynamoDBEndpoint);
+        this.dynamoDBClientConfig = config;
+
+        return this;
+    }
+
+    /**
```

Comment:
^ This is the only important part:
* https://github.com/lyft/amazon-kinesis-client/blob/lyft-v1.2.1/src/main/java/com/amazonaws/services/kinesis/multilang/MultiLangDaemon.java#L124
* https://github.com/lyft/amazon-kinesis-client/blob/lyft-v1.2.1/src/main/java/com/amazonaws/services/kinesis/clientlibrary/config/KinesisClientLibConfigurator.java#L139
* https://github.com/lyft/amazon-kinesis-client/blob/lyft-v1.2.1/src/main/java/com/amazonaws/services/kinesis/clientlibrary/config/KinesisClientLibConfigurator.java#L172-L173